core/src/main/java/org/mule/runtime/core/privileged/processor/MessageProcessors.java

```diff
@@ -617,7 +616,8 @@ public static ProcessingStrategyFactory getDefaultProcessingStrategyFactory(Mule
   }
 
   public static ProcessingStrategyFactory createDefaultProcessingStrategyFactory() {
-    return new TransactionAwareStreamEmitterProcessingStrategyFactory();
+    return new TransactionAwareProactorStreamEmitterProcessingStrategyFactory();
```

Comment:
is this ok?

Reply:
certainly not. I have a branching kilomb